### ISSUE TITLE
Error deploying Cockpit through AYS

### ISSUE THREAD

**Issue by yveskerwyn** [Jumpscale/jscockpit#65]
Check: ssh yves@85.255.197.109 -p 7122
PW: on-demand

ays blueprint -> succeeds
ays init -> succeeds
ays install -> fails

```
TRACEBACK:

*TRACEBACK*********************************************************************************

Traceback (most recent call last):
  File "/tmp/actions/ays_ays_cockpit/install.py", line 35, in install
    self.open_port(service, requested_port=ss[1], public_port=ss[0])
  File "/opt/code/ays_cockpit/recipes/node.ovc/actions.py", line 52, in open_port
    executor = j.tools.executor.getSSHBased(service.hrd.get("publicip"), service.hrd.getInt("sshport"), 'root')
  File "/opt/jumpscale8//lib/JumpScale/tools/executor/ExecutorFactory.py", line 55, in getSSHBased
    self._executors[key] = ExecutorSSH(addr, port=port, login=login, passwd=passwd, debug=debug, checkok=checkok, allow_agent=allow_agent, look_for_keys=look_for_keys, pushkey=pushkey,pubkey=pubkey)
  File "/opt/jumpscale8/lib/JumpScale/tools/executor/ExecutorSSH.py", line 27, in __init__
    self.sshclient.connectTest()
  File "/opt/jumpscale8//lib/JumpScale/clients/ssh/SSHClient.py", line 170, in connectTest
    rc, out = self.execute(cmd, showout=False)
  File "/opt/jumpscale8//lib/JumpScale/clients/ssh/SSHClient.py", line 204, in execute
    ch = self.transport.open_session()
  File "/opt/jumpscale8//lib/JumpScale/clients/ssh/SSHClient.py", line 118, in transport
    if self.client is None:
  File "/opt/jumpscale8//lib/JumpScale/clients/ssh/SSHClient.py", line 141, in client
    raise j.exceptions.RuntimeError('Impossible to create SSH connection to %s:%s' % (self.addr, self.port))
OurExceptions.RuntimeError: ERROR: Impossible to create SSH connection to 85.255.197.114:2200 ((type:runtime.error)
[Tue05 14:07] - ...lib/JumpScale/tools/actions/Action.py:794  - ERROR    -

******************************************************************************************

[Tue05 14:07] - ...tyourservice/ActionMethodDecorator.py:93   - ERROR    - **ERROR ACTION**:
action: install              runid:ays_ays_cockpit (ERROR)




action: run                  runid:85.255.197.114:2200:cloudscalers      (OK)
RESULT:
[Tue05 14:07] - ...lib/JumpScale/tools/actions/Action.py:794  - ERROR    -

******************************************************************************************


**** TRACEBACK ***
Traceback (most recent call last):
  File "/usr/local/bin/ays", line 330, in <module>
    cli()
  File "/usr/local/bin/ays", line 163, in install
    run.execute()
  File "/opt/jumpscale8/lib/JumpScale/baselib/atyourservice/AYSRun.py", line 276, in execute
    raise j.exceptions.RuntimeError(self.error)
OurExceptions.RuntimeError: ERROR: RUN:ays_cockpit 4
```

Blueprint used: 

```
Yves Kerwyn, [06.07.16 11:16]
g8client__main:
  g8.url: 'be-scale-1.demo.greenitglobe.com'
  g8.login: 'yves'
  g8.password: '****'
  g8.account: 'Account of Yves'

cockpit.g8__pilot:
  g8.client: 'main'
  telegram.token: '*****'
  cockpit.name: 'cloud space for cockpit of yves'
  dns.sshkey: '/root/.ssh/dns_key'
  ays.repo.url: 'https://github.com/yveskerwyn/cockpit_gig_yves'
  dns.domain: 'vreegoebezig.barcelona.aydo.com'
  oauth.client_secret: '*****'
  oauth.client_id: 'Vreegoebezig'
  oauth.organization: 'Green IT Globe'
  oauth.jwt_key: '-----BEGIN PUBLIC KEY-----MHYwEAYHKoZIzj0CAQYFK4EEACIDYgAES5X8XrfKdx9gYayFITc89wad4usrk0n27MjiGYvqalizeSWTHEpnd7oea9IQ8T5oJjMVH5cc0H5tFSKilFFeh//wngxIyny66+Vq5t5B0V0Ehy01+2ceEon2Y0XDkIKv-----END PUBLIC KEY-----'
  staging: true
```

**Comment by yveskerwyn** [comment]
Same problem as reported by @ islamTaha12: https://github.com/Jumpscale/jscockpit/issues/61

**Comment by zaibon** [comment]
Found two problems:
1. sshkey service had a workaround that misbehave when installing two sshkey service at the same time. see : https://github.com/Jumpscale/ays_jumpscale8/commit/4095f37d797feb44565b063953bd42b30020932d
2. last version of caddy didn't support staging environment. I change the buildling of caddy to use last version to work with staging environment. see : https://github.com/Jumpscale/jumpscale_core8/commit/53448293470d0b2094111b1f37f0980273f0e191

I updated the jumpscale/g8cockpit docker image to contain good version of caddy.
deployement work now, but the sshkey issue need to be tackle in a better way.
